config/cobblegenrandomizer-common.toml

```diff
@@ -29,5 +29,5 @@

```

Comment:
I mean, it's `scoria` not `scorchia`

Reply:
There is `scoria` and `scorchia` in the modpack tho :sweat_smile:

Reply:
Oh jeez